modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/opml/OpmlImportTask.kt

```diff
@@ -227,6 +240,27 @@ class OpmlImportTask @AssistedInject constructor(
         processUrls(urls)
         return urls.size
     }
+    private fun replaceInvalidXmlCharacter(inputStream: InputStream): InputStream {
+        val content = inputStream.bufferedReader().use { it.readText() }
```

Comment:
This reads the whole input stream to the memory. For large files it will cause OOM.

Reply:
[758972f](https://github.com/Automattic/pocket-casts-android/pull/2260/commits/758972fbd3eaf166d2c440c9f17816f29016f8be)